CMakeLists.txt

```diff
@@ -12,6 +12,14 @@ endif()
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (UNIX AND NOT APPLE)
```

Comment:
Please document the backgrounds of this logic. Is there a reason a "system breakpad" can't be used on non-Linux systems and vice versa?